virttest/ip_sniffing.py

```diff
@@ -268,5 +268,38 @@ def _output_handler(self, line):
             return
 
 
+class TSharkSniffer(Sniffer):
+
+    """
+    TShark sniffer class.
+    """
+
+    command = "tshark"
+    options = ("-npi any 'port 68 or port 546' -T fields -E header=y "
+               "-E separator=/s -E occurrence=f -e ip.src -e ip.dst "
+               "-e bootp.type -e bootp.id -e bootp.hw.mac_addr "
+               "-e bootp.ip.your -e bootp.option.dhcp")
```

Comment:
My version wireshark-cli-2.2.7-1.fc25.x86_64 does not allow named options after positionals, could you please re-arrange this so the `'port 68 or port 546'` is the last argument?

Reply:
Good catch, will fix that, thanks.